.github/workflows/tests.yml

```diff
@@ -337,16 +337,21 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        reco_algo: [
+          millipede_original,
+          millipede_wilks,
+          splinempe
+        ]
         eventfile: [
           hese_event_01.json,
           run00136766-evt000007637140-GOLD.pkl,
           run00136662-evt000035405932-BRONZE.pkl
         ]
-        reco_algo: [
-          millipede_original,
-          millipede_wilks,
-          # splinempe
-        ]
+        exclude:
```

Comment:
Would it be cleaner to just have it run on hese as part of the test?

Reply:
Yes. At the beginning I was using `OnlineL2_SplineMPE` as seed so it was not available for HESE events. Now I take `VHESelfVeto` so we could run on HESE. Will try to get this in before merging.

Reply:
OK, this is not possible straight away because `OnlineL2_SplineMPE` is still used as energy reco seed. I am going to move forward with the merge for now but if a solution comes up I am happy to follow up in a future PR.